File: Python Excel Examples/PropertiesApiDemo/setDocumentProperty.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.properties_api import PropertiesApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.properties_api.PropertiesApi(configuration)
name = "SetDocumentProperties.xlsx"
password = ""
folder = "Properties"
storage = ""
propertyName = "Author";
properties= spirecloudexcel.models.DocumentProperty("Author", "e-iceblue",True)
api.set_document_property(name, propertyName, _property = properties, password = password, folder = folder, storage = storage)

File: PHP Excel Examples/PropertiesApiDemo/setDocumentProperty.php
<?php

use Spire\Cloud\Excel\Sdk\Api\PropertiesApi;
use Spire\Cloud\Excel\Sdk\Configuration;
use Spire\Cloud\Excel\Sdk\Model\DocumentProperty;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new PropertiesApi($configuration);

$name = "SetDocumentProperty.xlsx";
$folder = "input";
$password = null;
$storage = null;
$propertyName = "Keywords";
$property = new DocumentProperty(array('name' => "Keywords", 'value' => "SetDocumentProperty_xlsx", 'built_in' => true));
$apiInstance->setDocumentProperty($name, $propertyName, $property, $password, $folder, $storage);
